src/ruby/meetings/app/models/means_of_influence.rb

```diff
@@ -2,15 +2,20 @@ class MeansOfInfluence < ApplicationRecord
   def self.types
     %w(Meeting Hospitality Gift Travel)
   end
-  validates :month,:year,presence: true
+  validates :month,:year,:source_file_id,:source_file_line_number,presence: true
   has_many :influence_government_office_people
   has_many :influence_organisation_people
-  has_many :organisations,through: :influence_organisation_people, source: 'organisation'
+  has_many :organisations,through: :influence_organisation_people
```

Comment:
This is a regression, effectively reverting e355524b3027b3c652ac01222b37d376ff81dda4.  I'm assuming that wasn't deliberate but correct me if I'm wrong.